src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -331,6 +331,16 @@
         </xsd:sequence>
     </xsd:complexType>
 
+    <xsd:simpleType name="default_middleware">
+        <xsd:restriction base="xsd:string">
+            <xsd:enumeration value="true" />
+            <xsd:enumeration value="false" />
+            <xsd:enumeration value="1" />
```

Comment:
Will `1` and `0` be transformed to `true` and `false`?

Reply:
yes, that's always the case in attributes thanks to `XmlUtils` in the Config component